src/content/blog/2024-07-30-open-payments-cinderella-story.mdx

```diff
@@ -72,7 +73,7 @@ This approach is starting to push the boundaries of convenience and ability to p
 
 An early attempt to add this context to an authorized payment was through something called “mandates”. These were objects that a third party client would create on an Open Payments resource server that contained the aforementioned payment information. That mandate would then be referenced inside of a Authorization Details object, which would then be stringified and passed as a query parameter in an OAuth authorization URL:
 
-```
+```bash wrap
```

Comment:
This seemed like the most straightforward solution: https://expressive-code.com/key-features/word-wrap/